dev/io.openliberty.microprofile.health.3.1.internal/src/io/openliberty/microprofile/health31/spi/impl/HealthCheck31ResponseProviderImpl.java

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
```

Comment:
Why does this class have a different copyright?

Reply:
Good eye! Updated to the latest IBM Copyright header. I think the old copyright was used in the earlier days.